OnLand/ControlLaptop/flaskConfig/flaskConfig.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+# TODO: add db stuff in future?
+class FlaskConfig:
+    
+    def __init__(self, json, db):

```

Comment:
Can we have this create the json file if it's not created to path `localStorage/` or a better name.

Reply:
address in 0105bdb